str0m/src/stats.rs

```diff
@@ -97,6 +99,8 @@ pub struct MediaIngressStats {
     pub rid: Option<Rid>,
     // total bytes received
     pub bytes: u64,
+    // total number of rtp packets received
+    pub packets: u64,
```

Comment:
This commit is a bit strange in isolation since it just adds struct fields without reading or writing to them.